src/ebmc/ebmc_base.cpp

```diff
@@ -522,8 +524,8 @@ int ebmc_baset::do_bmc(prop_conv_solvert &solver, bool convert_only) {
         const namespacet ns(symbol_table);
         CHECK_RETURN(trans_expr.has_value());
         ::unwind(*trans_expr, *message_handler, solver, bound+1, ns, true);
-        result=finish_bmc(solver);
-        #endif
+        result=finish_word_level_bmc(solver);
+#endif
```

Comment:
I'd suggest to also move the `#if 0` at this point.

Reply:
sure